vllm/engine/metrics.py

```diff
@@ -15,72 +14,51 @@
 
 logger = init_logger(__name__)
 
-disable_created_metrics()
```

Comment:
I do not know what this did, but the equivalent does not seem to exist for `aioprometheus`

Reply:
https://github.com/prometheus/client_python/blob/master/prometheus_client/metrics.py#L74

Reply:
https://prometheus.github.io/client_python/instrumenting/#disabling-_created-metrics